packages/keystone/apolloServerPlugins/index.js

```diff
@@ -6,4 +7,5 @@ module.exports = {
     ApolloQueryBlockingPlugin,
     ApolloRateLimitingPlugin,
     ApolloSentryPlugin,
+    ApolloPayloadLimitingPlugin,
```

Comment:
please add this logic inside `ApolloRateLimitingPlugin` it's enough plugins ...

Reply:
We'll merge them later I guess